spec/javascripts/app/utils/events_spec.js

```diff
@@ -0,0 +1,88 @@
+const Events = require('../../../../app/assets/javascripts/app/utils/events').default;
+/*eslint-disable */
+const sinon = require('sinon');
+/*eslint-enable */
+
+describe('Events', () => {
+  const myEvent = 'super.event';
+  const dummyHandler = sinon.stub();
+  let events;
+
+  beforeEach(() => {
+    events = new Events();
+  });
+
+  it('maintains a on object of to store handlers', () => {
```

Comment:
Oh yeah that is super unclear! I think mangled it by deleting parts of it and trying to rewrite it...